docs/CNAME

```diff
@@ -1 +0,0 @@
-u-root.tk
```

Comment:
Won't this break our web presence again?

Reply:
No idea. I put it back though. I just put whatever mkdocs did in the commit. *shrug*

Reply:
Yes, will break.

Reply:
@hugelgupf I read in another place that you bought the `u-root.com` domain, are you using that for something? Would be nice setting up it as `URL Forward` for `u-root.tk`. 

As alternative setting `u-root.com` as CNAME and pointing `u-root.tk` by URL Forward  will work too, but we'll lost some URI HTTP features.  Let's say that we'll use that setup because `u-root.com` seems more pleasant (I'm supposing that), so then the folders on URL: `u-root.tk/testing/etc/` will forward, naively without nginx reverse proxy, to `u-root.com` and maybe this will not be expected. 

I had a similar problem earlier days with my personal blog http://manoel.tk and http://lerax.me. I used for almost one year the domain http://manoel.tk, but for personal reasons I prefered migrate to http://lerax.me. Since I have the both domains (and hosted on gh-pages) I'd like that old links works as expected for any old url of http://manoel.tk that I pointed on somewhere. So for that I make some magic with nginx reverse proxy servers on DigitalOcean for given retro-compatibility with `manoel.tk` old domain routes, like `manoe.tk/lisp-chat` and etc. 

But it's a litte complex setup and I would prefer that things be more simpler for u-root, so forwarding `u-root.com` as `u-root.tk` probably will work well for us.